src/utils/api.js

```diff
@@ -27,6 +27,10 @@ class Api {
 		}
 	}
 
+	refreshToken() {
```

Comment:
@fcjr I'm thinking renaming this to something like `refreshTokenAndHandleIsRefreshing` to make it super clear what the purpose of this new function is

Reply:
That sounds reasonable, I'll make the change. My thought with just naming it `refreshToken` was that as an external user of the api class the inner "isRefreshing" state is an implementation detail that external users of the function don't really need to know about, but since this should only be used in special cases anyways there shouldn't be a harm in making it explicit

Reply:
@fcjr actually you do make a good point as it’s only used internally in the other “private” function. Ok you can just leave it as is then. Thanks.